config/routes.rb

```diff
@@ -1,4 +1,5 @@
 require "resque_web"
+ResqueWeb::Engine.eager_load!
```

Comment:
Oh, no. It's fine where it is. I just only read the diff and thought we might have forgotten to mount it. After looking at the whole file I realized my mistake.